charts/generic-service/ci/log-alerts-sidecar-values.yaml

```diff
@@ -0,0 +1,17 @@
+# log alerts test
+
+image:
+  repository: jwilder/whoami
+  tag: latest
+
+alerting:
+  enabled: true
+  logs:
+    levels:
+      warning:
+        severity: Warning
+        maxCount: 10
+      error: {}
+    lokiSideCar:
+      configMapLabel: "loki_rules"
+      configMapLabelValue: ""
```

Comment:
If `createLokiRule: true` is the default value, we'll need to set `createLokiRule: false` here.